packages/typegpu/src/data/align.ts

```diff
@@ -26,6 +26,8 @@ export class TgpuAlignedImpl<TAlign extends number, TData extends AnyTgpuData>
   implements TgpuAligned<TAlign, TData>
 {
   public readonly size: number;
+  isLoose = false as const;
```

Comment:
`align(N, ...)` overrides the "looseness" of a data-type, so the following can happen:

```ts
const foo = d.looseArray(vec3f, 4);

const bar = d.struct({
  a: d.u32,
  // Should fail, and fails, as `struct()` only accepts `TgpuData`, not `TgpuLooseData`
  b: foo,
});

const baz = d.struct({
  a: d.u32,
  // Should fail, but succeedes, as `align()` is always a `TgpuData`. This should differ based on the input to `align()`.
  b: d.align(16, foo),
});
```